broker-daemon/broker-rpc/wallet-service/commit-balance.js

```diff
@@ -71,9 +71,19 @@ async function commitBalance ({ params, relayer, logger, engines }, { EmptyRespo
     throw new PublicError(`Maxium balance of ${MAX_CHANNEL_BALANCE} exceeded for committing to the relayer. Please try again.`)
   }
 
+  const {maxBalance: maxOutboundBalance} = await engine.getMaxChannel()
+  const {maxBalance: maxInboundBalance} = await inverseEngine.getMaxChannel({outbound: false})
+  const convertedBalance = convertBalance(balance, symbol, inverseSymbol)
+  if (maxOutboundBalance || maxInboundBalance) {
```

Comment:
this is unneeded because we already check if these exist in the if statement below

Reply:
This is needed, because if neither of them exist then we open channels, but the conditions below determine what error we need to throw if either channel is open. But I guess if we break it up like you said then this is not necessary.